docs/Logging.md

```diff
@@ -98,3 +98,34 @@ fastify.get('/', function (request, reply) {
 ```
 
 *The logger instance for the current request is available in every part of the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md).*
+
+## Log Redaction
+
+[Pino](https://getpino.io) supports low-overhead log redaction, to
+filter out some of properties. As an example, if logging all the HTTP
+headers is needed, we must redact and filter out the `Authorization`
```

Comment:
I think the sentence is wrong because if we want to log all headers we do nothing.